### DISCUSSION TITLE
Spock Extension for Quarkus

### DISCUSSION THREAD

**Discussion by Andreas-Kreouzos** [spockframework/spock#1987] [General]
Hello,

my organization uses Spock extensively for testing all of the applications based on OpenLiberty with Microprofile. This occurs not only for Unit Testing but also for integration tests. However, we are planning to use Quarkus for some of our newer applications and unfortunately there's no support for Spock.

I see that there's an open issue [here](https://github.com/quarkusio/quarkus/issues/30221) that does not have any progress for now. So is there any possibility to create a new Spock extension for Quarkus?

Thank you in advance

**Comment by Vampire** [comment]
Anyone can write Spock extensions and also list them on https://github.com/spockframework/spock/wiki/Third-Party-Extensions, so you are welcome to write an own Quarkus integration extension if the work of @roggenbrot died away.

If you are asking whether the Spock team could create such an extension, that's quite unlikely. Resources are very limited, and probably noone even uses Quarkus. Otherwise I guess they would already have created such an extension.

See also #1832.

**Reply by Andreas-Kreouzos** [reply]
Thank you for the quick reply. I will try it.
